core/websock.js

```diff
@@ -1,12 +1,12 @@
 /*
- * Websock: high-performance binary WebSockets
+ * WebChannel: high-performance binary WebSocket / RTCDataChannel
```

Comment:
Probably time to fix this comment now and just call it a wrapper. :)

Reply:
`WebChannel: high-performance wrapper over a WebSocket or RTCDataChannel`?

Reply:
That or perhaps just "wrapper" or "buffering wrapper". The longer description further down seems to be more accurate.